### PR TITLE
[Storm-Compatibility] Forward Storm Kryo registrations to Flink

### PR DESCRIPTION
**Pull request by mjsax**

### CODE REVIEW

File: flink-contrib/flink-storm/src/main/java/org/apache/flink/storm/api/FlinkClient.java [hunk 1]
```diff
@@ -181,9 +188,11 @@ public void submitTopologyWithOpts(final String name, final String uploadedJarLo
...
 		}
 
-		/* set storm configuration */
-		if (this.conf != null) {
-			topology.getExecutionEnvironment().getConfig().setGlobalJobParameters(new StormConfig(this.conf));
+		try {
+			FlinkClient.addStormConfigToTopology(topology, conf);
+		} catch(ClassNotFoundException e) {
+			LOG.error("Could not register class for Kryo serialization.", e);
+			throw new InvalidTopologyException();
```
Comment (StephanEwen) [inline flink-contrib/flink-storm/src/main/java/org/apache/flink/storm/api/FlinkClient.java]: Would be good to add the logged error message also to the exception. Gives more instant feedback to the user what exactly went wrong.

File: flink-contrib/flink-storm/src/main/java/org/apache/flink/storm/api/FlinkClient.java [hunk 2]
```diff
@@ -325,4 +334,27 @@ private ActorRef getJobManager() throws IOException {
 				actorSystem, AkkaUtils.getLookupTimeout(configuration));
 	}
 
+	@SuppressWarnings({ "unchecked", "rawtypes" })
+	static void addStormConfigToTopology(FlinkTopology topology, Map conf) throws ClassNotFoundException {
+		if(conf != null) {
```
Comment (StephanEwen) [inline flink-contrib/flink-storm/src/main/java/org/apache/flink/storm/api/FlinkClient.java]: The majority of the Flink code uses spaces after keywords (if, for, ...). We have no strict checkstyle for that, but if we actually add a variant of Google code style, it helps when code is already in style.

**Comment by StephanEwen** [comment]
One comment about exception message, minor comment concerning code style.

Otherwise good to merge in my opinion.

**Comment by mjsax** [comment]
Will fix both and merge. Thanks.

**Pull request closed without merging**